go/ipfs/cs.go

```diff
@@ -26,37 +25,50 @@ import (
 	"github.com/ipfs/go-ipfs/repo/fsrepo"
 )
 
-var (
-	CurrentNode *core.IpfsNode
-
-	// Ugg, such a hack. Setting this to an integer >= 0 ** <= 9 will cause
-	// NewChunkStore to alter the api, http, and swarm ports for this IPFS node
-	// to get reset to a port ending in that digit.
-	NodeIndex int = -1
-)
-
 // Creates a new ChunkStore backed by IPFS.
 //
 // Noms chunks written to this ChunkStore are converted to IPFS blocks and
 // stored in an IPFS BlockStore.
 //
-// The name distinguishes this chunkstore on the local machine. A corresponding
-// file is created in ~/.noms/ipfs which stores the root of the noms database.
-// This should ideally be done with IPNS, but that is currently too slow to be
-// practical.
+// Noms repositories that are build on top of IPFS are identified by paths of
```

Comment:
IPFS database specs have the form: `ipfs://...`

Reply:
Done